app/controllers/api/base_controller.rb

```diff
@@ -148,6 +152,10 @@ def redirect_api_request(method)
       api_error_type(:not_found, "Unknown resource specified")
     end
 
+    def options # OPTIONS
```

Comment:
I think there shouldn't be the need to do the redirect anymore (i.e. this method body will become that of `options_generic`), `options_generic` can disappear, providing you call the method in the ArbitrationRulesController `#options` too